Tools/unicode/genmap_tchinese.py

```diff
@@ -0,0 +1,239 @@
+#
+# genmap_tchinese.py: Traditional Chinese Codecs Map Generator
+#
+# Original Author:  Hye-Shik Chang <perky@FreeBSD.org>
```

Comment:
FWIW @hyeshik doesn't seem to be active on this repo, but he signed the CLA in 2005 (https://bugs.python.org/user1298), so it should be ok to include it here.

If this script is based on his script, you might want to clarify that in the comment.